zwave_js_server/model/node.py

```diff
@@ -134,6 +134,7 @@ class NodeDataType(EndpointDataType):
     commandClasses: List[CommandClassInfoDataType]
     values: List[ValueDataType]
     statistics: NodeStatisticsDataType
+    highestSecurityClass: SecurityClass
```

Comment:
Is the type correct? Isn't this just the integer value?

Reply:
yes, fixing this here and in the other two places I made this mistake